css/style.css

```diff
@@ -16,4 +16,205 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* header */
+.header {
+    max-width: 1200px;
+    margin: 0 auto;
+    font-family:Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1rem;
+    padding: 1rem;
+    padding-left: 50px;
+}
+
+.main-logo {
+    height: 50px;
+    width: auto;
+}
+
+/* I've started trying to do mobile version and then found out that we don't need to do it now */
+.menu-hamburger {
+    display: none;
+    width: 30px;
+    height: 30px;
+}
+
+/* navigation */
+.nav-list {
+    display: flex;
+    flow-flex: row wrap;
```

Comment:
Should this be "flex-flow"? See this page: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow